src/Settings.tsx

```diff
@@ -178,6 +187,22 @@ export const Settings = (): JSX.Element => {
           />
         </>
       )}
+      {/* mantis data directory */}
+      <div className="settings-item">
+        <div className="settings-label">Mantis data directory</div>
+        <div className="settings-input select-folder">
+          <BorderlessInput disabled value={mantisDatadirInUse} />
```

Comment:
I can easily imagine that you won't be able to see the whole path here, because the input is quite small :) Even for my default datadir it's already trimmed. Not sure what would be the best way to address this though.

Reply:
Smaller font size? Ability to scroll the input?
I'll make some experiments, I definitely don't want to break the UI consistency.

Reply:
Sadly, I couldn't find a pure CSS solution. I fixed it by dynamic font size based on the length of the text.
Font size won't go below 8px, because it would be unreadable. If the text overflows in that case, it'll get ellipsized.